Shared/ForChildren/ForChildrenView.swift

```diff
@@ -8,13 +8,49 @@
 import SwiftUI
 
 struct ForChildrenView: View {
+    @EnvironmentObject var dataStore: ForKidsDataStore
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack {
+            if dataStore.forKids != nil {
```

Comment:
pro zjednodušení reference dataStore.forKids! udělej toto:

```
if let forKids = dataStore.forKids {
```

a pak 

```
ForEach(forKids) { item in
```

Tím že v if dáš let tak zaručíš že to není nil :)

Reply:
Thanks, fixed!